src/libsync/configfile.cpp

```diff
@@ -363,6 +363,17 @@ QString ConfigFile::configPath() const
     return dir;
 }
 
+QString ConfigFile::localConfigFile() const
+{
+    const QString settingspath = QStandardPaths::writableLocation(QStandardPaths::AppDataLocation);
```

Comment:
AppDataLocation is the wrong path here.
Use `QStandardPaths::ConfigLocation`.

Read the Qt docs for more:
https://doc.qt.io/qt-5/qstandardpaths.html#StandardLocation-enum